broadinstitute/pearl
# AR-204 adding lifestyle survey

Adds the family history survey from https://docs.google.com/document/d/1ct8grbqTu5F3HVAEsaVQflQ5F_iXZ28b/edit

This also adds back Markdown parsing of SurveyJS questions.  This uses micromark, which is the default parser for react-markdown, which we are already using for the landing page displays.  

TO TEST:
1. npm install
2. repopulate OurHealth
3. sign into participant view as `consented@test.com`
4. take "Lifestyle" survey, confirm it generally behaves
